app/docs/src/how-tos/aws-lambda.md

```diff
@@ -0,0 +1,408 @@
+`Role ARN` input socket of this `HelloWorldNodeFunction` component.
+
+In order to ensure that you are setting the correct version of the container
+image, you can inspect the docker container manifest to ensure you set the
+correct image to use as the container image. My local setup will try and build
+multi-architecture containers, so I need to select the correct `arm64`
+architecture for the function.
+
+```bash
+docker manifest inspect 573113295293.dkr.ecr.us-east-1.amazonaws.com/lambda-how-to-repository:latest
+{
+   "schemaVersion": 2,
+   "mediaType": "application/vnd.oci.image.index.v1+json",
+   "manifests": [
+      {
+         "mediaType": "application/vnd.oci.image.manifest.v1+json",
+         "size": 1626,
+         "digest": "sha256:703e717188a34c38e53d80a3dec4bf345c5b9b97a4f40648aa2987e296e099a5",
+         "platform": {
+            "architecture": "arm64",
+            "os": "linux"
+         }
+      },
+      {
+         "mediaType": "application/vnd.oci.image.manifest.v1+json",
+         "size": 566,
+         "digest": "sha256:e425754d9a0571c59a7a138eb4989b5cdcdc7153d2c2dbd9b47f4adc0b91cb8d",
+         "platform": {
+            "architecture": "unknown",
+            "os": "unknown"
+         }
+      }
+   ]
+}
+```
+
+In this case, I would set the `ContainerImageVersion` to be
+`@sha256:703e717188a34c38e53d80a3dec4bf345c5b9b97a4f40648aa2987e296e099a5`.
+
+### Create Lambda Permission
+
+![Create Lambda Permission](./aws-lambda/create-lambda-permission.png)
+
+Add a `Lambda Permission` to your `HelloWorldNodeFunction` Lambda function
+frame.
+
+Set the component name to be `FunctionPublicPermission`.
+
+Set `Action` to be `lambda:InvokeFunctionUrl`
+
+Set `FunctionUrlAuthType` to be `NONE`.
+
+Set `Principal` to be `*`.
+
+Set `StatementId` to be `AllowPublicAccess`.
+
+### Create Lambda Function Url
+
+![Create Lambda Function URL](./aws-lambda/create-lambda-function-url.png)
+
+Add a `Lambda Function Url` to your `HelloWorldNodeFunction` Lambda function
+frame.
+
+Set the component name to be `FunctionPublicUrl`
+
+Set the `AuthType` to be `NONE`.
+
+### Apply your Change Set
+
+Press `Escape` or click anywhere on the canvas background to select the
+Workspace.
+
+Click the `Apply Change Set` button to:
+
+- Create 2 IAM Roles and the correct attachment between them
+- Create a Lambda Function
+- Create a Lambda Permission
+- Create a Lambda Function URL
+
+### Explore your resources
+
+Review the completed AWS resources by clicking the `Resource` sub-panel for each
+of your new resources.
+
+Select the `FunctionPublicUrl` component on the model. Go to the `Resource Data`
+screen and you will find the `FunctionUrl`. You can visit that URL in the
+browser and inspect the running application.
+
+### Clean Up
+
+Create a new change set called `Clean up How-to`
+
+Delete all of the components.
+
+Click `Apply Change Set`.
+
+All your new resources should be deleted from your AWS account.
```

Comment:
Might be worth noting locally they can delete the directory of content you made them create earlier

Reply:
I am hoping they check it into source control ;)